pkg/arvo/sys/arvo.hoon

```diff
@@ -1114,13 +1194,13 @@
       ?:  ?=(%$ cyr)
         (peek:pith lyc %$ bem)
       ::
-      ::  XX vane and care are concatenated
+      ::  vane and care are concatenated
       ::
-      =/  lal  (end 3 cyr)
-      =/  ren  ;;(@t (rsh 3 cyr))
-      ?.  (~(has by van.mod) lal)
+      =/  way=term  (grow (end 3 cyr))
+      =/  car=term  (rsh 3 cyr)
```

Comment:
Don't you want this to support `%clayx` style vane+care?

Reply:
I'm not quite sure how to generically support all of `%$`, `%j`, `%cx`, %clayx`.

Reply:
Also, `%clayx` is *really* ugly.

Reply:
But, this current API doesn't support non-abbreviated names, which is a problem. Arguably, `(room)` should take `[gang route care beam]`, and this combined vane/care parsing should go into `+look`. I'll think about it.

Reply:
Note that `|mass` is currently broken on this branch due to exactly this problem.

Reply:
```
?=([~ %many [[%$ %tas @] [%$ %tas @] ~]] (rush ~.._ames_x__ ;~(pfix dot perd:so)))
```

Reply:
To be clear, I'm not wed to the `%clayx` format in particular, my concern is indeed about the non-abbreviated names that are now supported throughout the rest of arvo.

I think you _can_ discriminate between the term variants you gave, but you'd be making up janky-feeling restrictions like "only care-less scries using vane abbreviations". Using `%many` for this could be fun!